google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -374,18 +375,19 @@ private void validateConfiguration() throws ConfigurationException {
     if (!isValidConfigurationFile(getAppYamlPath())) {
       throw new ConfigurationException(
           GctBundle.getString("appengine.deployment.error.staging.yaml") + " "
-              + GctBundle.getString("appengine.deployment.error.staging.gotosettings"));
+              + GctBundle.getString("appengine.deployment.error.staging.file.gotosettings"));
```

Comment:
Similar comment here as for the Docker directory messaging: this just mentions an "invalid file", not a bad app.yaml which is why the error is shown.

Aren't these messages shown at the bottom of the deployment config? If so they should mention what they are for.